### PR TITLE
Add get statistics per aggregation unit

### PR DESCRIPTION
**Pull request by hinoguma**
#350

### CODE REVIEW

File: src/LINEBot.php [hunk 1]
```diff
@@ -249,23 +259,29 @@ public function pushMessage($to, MessageBuilder $messageBuilder, $notificationDi
...
+     * @param array|null $customAggregationUnits Name of aggregation unit.
      * @return Response
      */
     public function multicast(
         array $tos,
         MessageBuilder $messageBuilder,
         $notificationDisabled = false,
-        $retryKey = null
+        $retryKey = null,
+        $customAggregationUnits = null
```
Comment (Vaduz) [inline src/LINEBot.php:270]:
```suggestion
        array $customAggregationUnits = array()
```

File: src/LINEBot.php [hunk 2]
```diff
@@ -227,19 +227,29 @@ public function replyText($replyToken, $text, $extraTexts = null)
...
      * @return Response
      */
-    public function pushMessage($to, MessageBuilder $messageBuilder, $notificationDisabled = false, $retryKey = null)
-    {
+    public function pushMessage(
+        $to,
+        MessageBuilder $messageBuilder,
+        $notificationDisabled = false,
+        $retryKey = null,
+        $customAggregationUnits = null
```
Comment (Vaduz) [inline src/LINEBot.php:238]:
```suggestion
        array $customAggregationUnits = array()
```

File: src/LINEBot.php [hunk 3]
```diff
@@ -1251,4 +1267,52 @@ public function testWebhookEndpoint($endpoint)
...
+     */
+    public function getNameListOfUnitsUsedThisMonth($limit = null, $start = null)
+    {
+        $data = [];
+        if (isset($limit)) {
+            $data['limit'] = $limit;
+        }
+        if (isset($start)) {
+            $data['start'] = $start;
+        }
```
Comment (Vaduz) [inline src/LINEBot.php:1315]: Please rewrite continuation token implementation.
Using this token, you can get all remaining data from server.
See: https://github.com/line/line-bot-sdk-php/blob/210d87af3f5fa3ed7837c3c875e39e1e3caeb9b7/src/LINEBot.php#L109-L140

File: tests/LINEBot/AggregationUnitTest.php [hunk 4]
```diff
@@ -0,0 +1,190 @@
+<?php
+
+/**
+ * Copyright 2020 LINE Corporation
```
Comment (Vaduz) [inline tests/LINEBot/AggregationUnitTest.php:4]:
```suggestion
 * Copyright 2021 LINE Corporation
```

Comment (hinoguma) [reply tests/LINEBot/AggregationUnitTest.php]: Thank you for suggestion.
Fixed them.

File: src/LINEBot.php [hunk 5]
```diff
@@ -1251,4 +1267,75 @@ public function testWebhookEndpoint($endpoint)
...
+     */
+    public function getNumberOfUnitsUsedThisMonth()
+    {
+        return $this->httpClient->get($this->endpointBase . '/v2/bot/message/aggregation/info');
+    }
+
+    /**
+     * Get the name list of units used this month for statistics aggregation.
+     *
+     * @param string|null $limit
```
Comment (moririnson) [inline src/LINEBot.php:1303]: int|null

**Review by Vaduz — APPROVED** [review]

**Comment by moririnson** [comment]
@hinoguma thanks!!

**Pull request merged**